src/__tests__/InputField.js

```diff
@@ -24,6 +24,105 @@ describe('InputField', () => {
     )
     console.error.mockRestore()
   })
+  test('it renders using FormField alias', () => {
+    const json = renderer
+      .create(
+        <FormField onChange={() => {}}>
+          <Icon name="email" />
+          <Input id="with-left-icon" />
+        </FormField>
+      )
+      .toJSON()
+    expect(json).toMatchSnapshot()
+  })
+
+  test('it renders using FormField alias with Select ', () => {
+    const json = renderer
+      .create(
+        <FormField onChange={() => {}}>
+          <Select id="with-no-options" />
+        </FormField>
+      )
+      .toJSON()
+    expect(json).toMatchSnapshot()
+  })
+
+  test('it renders using FormField alias with Select and Icon', () => {
+    const json = renderer
+      .create(
+        <FormField onChange={() => {}}>
+          <Icon name="email" />
+          <Select id="with-left-icon" />
```

Comment:
Tiny test case nit: seems the test is testing "_left icon_" & "_no options_" in the composition here, could the `id` value be something that combines the essence of two characteristics of this unit test?

Reply:
Sure it can... I'll work on that.